kubetest/client.py

```diff
@@ -215,6 +216,48 @@ def load_service(self, path, set_namespace=True):
             service.namespace = self.namespace
         return service
 
+    def load_PersistentVolume(self, path, set_namespace=False):
+        """Load a manifest YAML into a Service object.
+
+        By default, this will augment the Service object with
+        the generated test case namespace. This behavior can be
+        disabled with the ``set_namespace`` flag.
+
+        Args:
+            path (str): The path to the Service manifest.
+            set_namespace (bool): Enable/disable the automatic
+                augmentation of the Service namespace.
+
+        Returns:
+            objects.Service: The Service for the specified manifest.
```

Comment:
This help block needs to be updated to reflect PVs :)

Reply:
Thanks for your feedback. I have fixed based on your commentaries.